custom_components/opnsense/switch.py

```
@@ -524,22 +543,30 @@ async def async_turn_on(self, **kwargs: Any) -> None:
-            await self.coordinator.async_refresh()
+        async def start_service():
+            assert self._service is not None
+            result: bool = await self._client.start_service(
+                self._service.get("id", self._service.get("name", None))
+            )
+            if result:
+                await self.coordinator.async_refresh()
+
+        await queue.put(start_service)
 
     async def async_turn_off(self, **kwargs: Any) -> None:
```

Pylint (3.3.7)

[warning] Unused argument 'kwargs'

(W0613)